portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IdentityEvents.testcase

```diff
@@ -814,4 +976,121 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-11127 | Automation ID: LRAC-12155 | Test Summary: New user changes emailAddressHashed"
+	@ignore = "true"
+	@priority = "5"
+	test NewUserChangeEmailAddressHashed {
+		property analytics.cloud.upstream = "false";
+		property proxy.server.enabled = "true";
+
+		var usernameList = "user1,user2";
+
+		task ("Add two new users") {
+			for (var username : list "${usernameList}") {
+				JSONUser.addUser(
+					userEmailAddress = "${username}@liferay.com",
+					userFirstName = "${username}",
+					userLastName = "${username}",
+					userScreenName = "${username}");
+
+				JSONUser.setFirstPassword(
+					agreeToTermsAndAnswerReminderQuery = "true",
+					requireReset = "false",
+					userEmailAddress = "${username}@liferay.com");
+			}
+		}
+
+		task ("Sync the Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Start Har recording") {
+			ProxyUtil.startHarRecording("identity");
+		}
+
+		task ("Sign in as new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "user1@liferay.com",
+				userLoginFullName = "user1 user1");
+		}
+
+		task ("Visit the document") {
+			ACUtils.navigateToSitePage(
+				actionType = "View DM",
+				documentTitleList = "DM AC Title",
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Wait for sending requests") {
+			ACUtils.waitForSendingRequest();
+		}
+
+		task ("Get the emailAddressHashed number of the identity event") {
+			var emailAddressHashed1 = ACUtils.getPageRefRequestPropertyValue(
+				pageRef = "identity",
+				property = "emailAddressHashed");
+		}
+
+		task ("Check that the emailAddressHashed is not empty") {
+			TestUtils.assertNotEquals(
+				actual = "${emailAddressHashed1}",
+				expected = "");
+		}
+
+		task ("Clear local storage") {
+			ACUtils.clearLocalStorage();
```

Comment:
Hey @ViniLopes20 , I'm wondering the purpose of clearing local storage here.

Reply:
But when we clear the local storage, then the userId will be changed when log in with another user. That way `user1` will not cover `user2` in the AC list